NAMESPACE

```diff
@@ -48,11 +53,16 @@ export(validateNumericCols)
 exportClasses(DataShape)
 exportClasses(DataType)
 exportClasses(PlotReference)
+exportClasses(Range)
```

Comment:
S4 is pretty formal, which i think for us will end up being a good thing. but it means that it makes a distinction between the class and what is essentially its constructor. so i could, when defining a class, just do something like `setClass(x)` rather than `x <- setClass(x)`. in that case youd see the exportClasses entry but not the regular export. then to make an instance of the class youd instead have to use the like general S4 constructor, something like `new('x')`

Reply:
ooooooh i like it!!!!!